frontend/src/components/ingredient/Ingredient.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from "react";
+import Recipe from "../recipe/Recipe";
+import AllRecipes from "../allRecipes/AllRecipes";
+import Spinner from "../spinner/spinner";
+import IngredientList from "../ingredientList/IngredientList";
+
+const Ingredient = ({ navigate }) => {
+  const [checked, setChecked] = useState([]);
+  const [matchedRecipes, setMatchedRecipes] = useState([]);
+  const [searchIngredients, setSearchIngredients] = useState([]);
+  const [collapse, setCollapse] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [unchecked, setUnchecked] = useState(true);
+
+  const FetchData = () => {
+    setLoading(true);
+    fetch(
+      "https://westeurope.azure.data.mongodb-api.com/app/recipe_api-eixns/endpoint/recipes"
```

Comment:
Just wanted to say to all involved (especially Max) well done on getting this sorted, hard graft.

In the next sprint we can look at finessing this fetch request, so the FE isn't dealing with all info on all recipes; can we not populate the fetch URL with the elements of the checked array with string interpolation?